lib/matplotlib/text.py

```diff
@@ -240,7 +240,8 @@ def update(self, kwargs):
         """
         bbox = kwargs.pop('bbox', None)
         super(Text, self).update(kwargs)
-        self.set_bbox(bbox)  # depends on font properties
+        if bbox:
```

Comment:
probably to be safe, this should be `if bbox is not None:`. I can't be sure if bbox is ever a numpy array or not, and doing a test like that on a numpy array will result in errors in some versions, I think.